frontend/package.json

```diff
@@ -23,7 +23,7 @@
     "sass": "^1.49.9"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
there is no need for this, please remove. If you have issues with installing dependencies you should change your node version to v16.x.x. There is currently undergoing story with this issue.